Assets/Scripts/Items/ItemObject.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/*
+    Nadklasa dla przedmiotów.
```

Comment:
I'm not sure I get it 😢
Also probably should be in english to stay consistent (I guess?)